tests/test_operations.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+# ignoring E402: module level import not at top of file
+os.environ['TMA_CERTIFICATE'] = 'cert content'  # noqa: E402
+os.environ['BRP_APPLICATIE'] = 'mijnAmsTestApp'  # noqa: E402
+os.environ['BRP_GEBRUIKER'] = 'mijnAmsTestUser'  # noqa: E402
+os.environ['MKS_BRP_ENDPOINT'] = 'https://example.com'  # noqa: E402
```

Comment:
Seems like the wrong way to do this. What happens if another test runs after this one? Maybe put it in a before each and restore situatiuon in an after each? Or try some mocking setup

Reply:
This needs to be set, otherwise the import of the flask app will error. There is a check if all the environment vars are set.